src/components/wb-playground/kyn-linked-div-wb/kyn-linked-div-wb.stories.js

```diff
@@ -0,0 +1,45 @@
+import { html } from 'lit';
+import './index';
+
+export default {
+  title: 'Components/KynLinkedDivWb',
```

Comment:
Use a friendly name instead of `KynLinkedDivWb` here. This is what will show in the Storybook sidebar

Reply:
i used this strange naming convention so that it would be obvious which ones where my playground components. i've changed them to more usable names for the Storybook sidebar now